service/topology_coordinator.cc

```diff
@@ -848,8 +847,8 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
                     break;
```

Comment:
looks like this break duplicates the break below now.

btw the whole `handle_global_request` code is a nightmare to look at